enhancements/installer/kubevirt-ipi.md

```diff
@@ -0,0 +1,338 @@
+---
+title: KubeVirt-platform-provider
+authors:
+  - "@ravidbro"
+reviewers:
+
+approvers:
+
+creation-date: 2020-07-14
+last-updated: 2020-07-14
+status: implementable
+---
+
+# KubeVirt-platform-provider
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [OpenShift/docs]
+
+## Open Questions [optional]
+
+## Summary
+
+This document describes how `KubeVirt` becomes a platform provider for OpenShift. \
+`KubeVirt` is a virtualization platform running as an extension of Kubernetes. \
+ We want to create a tenant cluster on top of existing OpenShift/Kubernetes cluster by creating
+ virtual machines by KubeVirt for every node in the tenant cluster (master and workers nodes)
+ and other Openshift/Kubernetes resources to allow **users** (not admins) of the infra cluster
+ to create a tenant cluster as it was an application running on the infra cluster.
+ To achieve that we will implement all the components needed for the installer and cluster-api-provider
+ for the machine-api to allow day 2 operations of resizing the cluster.
+
+
+## Motivation
+
+- Achieve true multi-tenancy of OpenShift were each tenant has dedicated control plane \
+ and have full control on its configuration. 
+
+### Goals
+
+- provide a way to install OpenShift on KubeVirt infrastructure using 
+  the installer - an IPI installation. (1st day)
+- implementing a cluster-api provider to provide scaling and managing the cluster
+  nodes (used by IPI, and useful for UPI, and also node management/fencing) (2nd day)
```

Comment:
+1, common verb-age is Day-1 and Day-2.

Reply:
I'd actually prefer to continue trying to get away from using "day 1" and "day 2" terms.  I don't find them very helpful.  I prefer "install-time" and "post-install".

cc @crawford since we were just talking about this